src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -82,21 +84,29 @@ public class Variables {
 	 */
 	private static final String CONFIGURATION_SERIALIZABLE_PREFIX = "ConfigurationSerializable_";
 
-	private final static Multimap<Class<? extends VariablesStorage>, String> TYPES = HashMultimap.create();
+	private final static Multimap<Class<? extends VariableStorage>, String> TYPES = HashMultimap.create();
+
+	private final static List<UnloadedStorage> UNLOADED_STORAGES = new ArrayList<>();
 
 	// Register some things with Yggdrasil
 	static {
-		registerStorage(FlatFileStorage.class, "csv", "file", "flatfile");
-		registerStorage(SQLiteStorage.class, "sqlite");
-		registerStorage(MySQLStorage.class, "mysql");
+		SkriptAddon source = Skript.getAddonInstance();
```

Comment:
Once you switch the SkriptAddon class being used, you'll want to use `Skript.instance()` as the source.